### DISCUSSION TITLE
Display the data

### DISCUSSION THREAD

**Discussion by DomiSanNo1** [pkuehnel/TeslaSolarCharger#1107] [Q&A]
Hey guys, is there a possibility to read out the data, displayed in TSC, for further project. E.g. display in OpenHab?

![image](https://github.com/pkuehnel/TeslaSolarCharger/assets/99594547/b4a2857e-7545-4335-b2a0-9e6da34370c1)

**Comment by pkuehnel** [comment]
Yes, TSC is completely based on REST APIs. Just add `/swagger` to the URL and you see all endpoints.

**Reply by DomiSanNo1** [reply]
Perfect! Thank you!
